core/src/utilities/helpers/iframe-helpers.js

```diff
@@ -285,9 +285,13 @@ class IframeHelpersClass {
   disableA11yOfInactiveIframe(srcIframe) {
     const nodeList = document.querySelectorAll('*');
     [...nodeList].forEach(el => {
-      el.setAttribute('oldTab', el.getAttribute('tabindex'));
-      if (el !== srcIframe) {
-        el.setAttribute('tabindex', '-1');
+      if (el.hasAttribute('tabindex')) {
```

Comment:
Code looks good and working as expected. Could you check if you can add a unit test for this one. I remember there is a similar one [here](https://github.com/SAP/luigi/blob/master/core/test/utilities/helpers/iframe-helpers.spec.js#L419)